### ISSUE TITLE
Missing SMPL-X parameters in test set

### ISSUE THREAD

**Issue by amcs-echo-0** [SangHunHan92/2K2K#37]
Hi, thanks for your great work and for providing this dataset.
I noticed that in the training set, the SMPL-X parameters are provided, but in the test set only the .ply files are available.

I would like to ask:

Should I obtain the SMPL-X parameters from another source?

Or is it expected that we fit the SMPL-X model ourselves from the provided .ply meshes?

Thanks in advance for your clarification!
